app/schemas/user.py

```diff
@@ -29,7 +29,14 @@ class UserSchema(Schema):
     age = fields.Method("get_age", dump_only=True)
     is_beta_user = fields.Boolean()
     credits = fields.Nested(CreditSchema, many=True, dump_only=True)
-
+    organisation = fields.String(required=True, error_message={
```

Comment:
This was made nullable in the database, yet we have it as required here, lets have a consistent constraint